pandas/tests/io/json/test_pandas.py

```diff
@@ -35,6 +35,15 @@
 _mixed_frame = _frame.copy()
 
 
+def strio_lines_json_df():
+    df = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]})
+    return StringIO(df.to_json(lines=True, orient="records"))
+
+
+def json_lines_to_df_chunked(jlines, chunksize):
```

Comment:
I wouldn't make this a function at all, just in-line it, its very confusing to the flow otherwise